### ISSUE TITLE
4.3 release article on mobile

### ISSUE THREAD

**Issue by JohnVeness** [godotengine/godot-website#904]
Many congratulations on the 4.3 release article. Overall it looks and reads great.

Just one nitpick for when viewing on mobile: Because the text all appears in one column (rather than two columns as on desktop), the choice of having pictures sometimes above the text and sometimes below the text in a section, can be a bit jarring. In particular, this can result in one picture appearing directly below another - the pictures are in different sections, but this is not immediately obvious, so it seems the pictures are related, causing momentary confusion. On desktop, the side-by-side nature makes it easy to see which pictures are from which section.

E.g.:

Mobile:
![Screenshot_20240816_165148_Firefox](https://github.com/user-attachments/assets/02200c11-2495-469c-a7fd-e7501948f693)
Desktop:
![image](https://github.com/user-attachments/assets/74fd5fdd-5136-4d25-bc34-861aebb963ac)

Mobile:
![Screenshot_20240816_165209_Firefox](https://github.com/user-attachments/assets/22ab60e7-d08e-4c03-91b9-a6b7c77dbbd2)
Desktop:
![image](https://github.com/user-attachments/assets/5b875603-f84e-4976-bef2-f3d57c156a4f)

This is probably too much work to fix quickly in this release, but maybe should be remembered if the same sort of template is used in a future release article. Thanks for reading :)

**Comment by coppolaemilio** [comment]
I agree. Maybe in the mobile view, it should have all the images always on top

**Comment by JohnVeness** [comment]
> I agree. Maybe in the mobile view, it should have all the images always on top

Yep. Or always on bottom. Either way, be consistent to avoid jarring the reader!

**Comment by coppolaemilio** [comment]
We are not going to be updating this page any time soon, but we'll consider this for the newer releases. Thanks!
